locales/base/translation.json

```diff
@@ -1268,6 +1268,24 @@
       "transactionDetailsAmountConverted": "Amount Withdrawn"
     }
   },
+  "fiatConnectStatusScreen": {
+    "tryAgain": "Try Again",
+    "withdraw": {
+      "cancel": "Cancel Withdrawal",
+      "success": {
+        "header": "Success",
+        "title": "Your funds are on their way!",
+        "description": "Your transaction request has been sent & your funds will arrive in [3-10] business days.",
```

Comment:
I think that's a slightly larger issue that would require designs/copy (due to the wide range of settlement times we could receive); I'm following the design verbatim here with the `3-10` copy, but happy to change it if there's something more up-to-date.

Reply:
Would it require that much of a refactor? You could use [getTimeEstimation() from FiatConnectQuote](https://github.com/valora-inc/wallet/blob/main/src/fiatExchanges/quotes/FiatConnectQuote.ts#L119). It would be good to be consistent between the select provider screen and this screen

Reply:
Ah, no it wouldn't be a big refactor, just a decent amount of logic to actually implement the `getSettlementEstimation` function correctly/get it to cover all common cases. That said, I've updated the implementation/copy here to use the `getTimeEstimation` method!

Reply:
Should there be a placeholder here for the timeEstimation param?

Reply:
omg, i forgot to _save_ the translation file after making the change :facepalm: